packages/lwt-dllist/lwt-dllist.1.1.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: [ "Anil Madhavapeddy <anil@recoil.org>" ]
+authors: ["Jérôme Vouillon" "Jérémie Dimino"]
+license: "MIT"
+homepage: "https://github.com/mirage/lwt-dllist"
+doc: "https://mirage.github.io/lwt-dllist/"
+bug-reports: "https://github.com/mirage/lwt-dllist/issues"
+depends: [
+  "ocaml" {>= "4.02.0"}
+  "lwt" {with-test}
+  "dune"
```

Comment:
Our lints generally require a lower bound on dune but I see the project builds with dune 1 (https://github.com/mirage/lwt-dllist/blob/master/dune-project) so I think this should be fine.